.ci/jenkins/Jenkinsfile.buildchain

```diff
@@ -101,15 +101,14 @@ pipeline {
         }
         stage('Sonar analysis') {
             tools {
-              jdk "kie-jdk11"
+              jdk "kie-jdk17"
               maven 'kie-maven-3.8.1'
             }
             steps {
                 script {
                     if(isPR()) {
                         def project = util.getProjectTriggeringJob()[1]
-                        if(["optaplanner", "drools", "appformer", "jbpm", "drools-wb", "kie-soup", "droolsjbpm-integration", "kie-wb-common", "optaweb-employee-rostering", "optaweb-vehicle-routing"].contains(project))
-                        {
+                        if(["optaplanner", "drools", "appformer", "jbpm", "drools-wb", "kie-soup", "droolsjbpm-integration", "kie-wb-common", "optaweb-employee-rostering", "optaweb-vehicle-routing"].contains(project)) {
```

Comment:
why we have here in the list optaplanner and optaweb repos? shouldn't were they removed

Reply:
yes - will remove them
cherry-pick problem!
good spot